lib/ansible/modules/cloud/amazon/dms_replication_subnet_group.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: dms_replication_subnet_group
+short_description: creates or destroys a data migration services subnet group
+description:
+    - creates or destroys a data migration services subnet group
```

Comment:
This should be regular text.
```suggestion
    - Creates or destroys a data migration services subnet group.
```